itest/lib/appStep/api/reload.js

```diff
@@ -0,0 +1,8 @@
+/* @flow */
+import {Application} from "spectron"
+
+import logStep from "../util/logStep"
+
+export default async (app: Application) => {
+  await logStep("app reload", () => app.browserWindow.reload())
+}
```

Comment:
When the return value of a function is a promise, and you don't need to capture a promise fulfillment in a variable, the async/await keywords are redundant.

This could be written like this:

```js
export default (app) => {
  return logStep("app reload", () => app.browserWindow.reload())
}
```

Whether you keep in the way it is, or change it to the above, the caller will probably "await" in the test file.

```js
await reload()
```

Reply:
This is going to get attention as part of #878